editor/plugins/script_editor_plugin.cpp

```diff
@@ -72,6 +72,7 @@
 #include "script_text_editor.h"
 #include "servers/display_server.h"
 #include "text_editor.h"
+#include <core/string/fuzzy_search.h>
```

Comment:
```suggestion
#include "core/string/fuzzy_search.h"
```
And it should be moved up to be sorted, like in #105239

Reply:
Ehhh, this is very weird. This was auto added by VSCode and normally get's sorted by the pre-commit hook, not sure why that didn't happen here. Well either way fixed it :).